src/components/Navbar/index.tsx

```diff
@@ -4,15 +4,46 @@ import Toolbar from "@material-ui/core/Toolbar";
 import { Typography } from "@mui/material";
 import { FunctionComponent } from "react";
 import Drawer from "../Menu";
```

Comment:
sort imports by where you're importing them from, e.g. first group will be of import from packages, then components, etc etc